app/views/entities/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="login-details">
+  <%= link_to groups_path do %>
+   <span class="fa-solid fa-chevron-left">
+    </span>
+  <% end %>
+<h2 class='login-header'>Transactions</h2>
+      <%= link_to "Sign-out", destroy_user_session_path, class: 'actions' %>
+</div>
+
+  <% @group.entities.each do |entity| %>
```

Comment:
- [ ] The transactions in each group should be ordered by the most recent. Kindly fix that. To avoid implementing logic in views I would recommend you refactor to this:

  ```
  <% @entities.each do |entity| %>
    ...
  ```
  Then in the index action of the entities controller you can defined the `@entities` variable like so:

  ```
  # EntitiesController

  def index
    @group = current_user.groups.find_by(id: params[:group_id])
    @entities = @group.entities.order(created_at: :desc)
  end
  ```

Reply:
Thanks for this awesome suggestion.